pypit/ardeimos.py

```diff
@@ -157,3 +157,41 @@ def deimos_read_1chip(hdu,chipno):
 
     # Return
     return data, oscan
+
+def bpm(det):
```

Comment:
Instead of having a function to do this, is it worth having these bad pixel masks stored as images in the repo?  It's certainly more storage efficient like this, but I like the flexibility of having the bad-pixel mask being read from a file instead, so that it can be changed/updated as needed by the user.

Reply:
Agreed.   Have raised this as Issue #331